### ISSUE TITLE
Getting Geotag of tweets using snscraper

### ISSUE THREAD

**Issue by koshyrani** [JustAnotherArchivist/snscrape#457]
How to get geotags and user locations of Twitter tweets using snscraper?

**Comment by JustAnotherArchivist** [comment]
Tweets have `place` and `coordinates` attributes, users have a `location` attribute.

**Comment by koshyrani** [comment]
> 

Thank you for the reply. Can we get the place and coordinate attribute using snscrape library. I dont have the developer version.

**Comment by JustAnotherArchivist** [comment]
You mean from Python?

```python3
for tweet in snscrape.modules.twitter.TwitterSearchScraper('...').get_items():
	if tweet.place:
		print(tweet.place)
	if tweet.user.location:
		print(tweet.user.location)
	#...
```

It's available since version 0.4.0.

**Comment by koshyrani** [comment]
Ok,Sir.  Thank you so much.
